framework/include/utils/RankFourTensor.h

```diff
@@ -84,13 +84,19 @@ class RankFourTensor
   static RankFourTensor IdentityFour() { return RankFourTensor(initIdentityFour); };
 
   /// Gets the value for the index specified.  Takes index = 0,1,2
-  Real & operator()(unsigned int i, unsigned int j, unsigned int k, unsigned int l);
+  inline Real & operator()(unsigned int i, unsigned int j, unsigned int k, unsigned int l)
```

Comment:
The `inline` keyword is not needed when you define the function in the class body